### PR TITLE
Fixed Demo variable and indentation! 

### PR DESCRIPTION
**Pull request by mcmchris**

### CODE REVIEW

File: IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Ubidots client(TOKEN);
+
+/****************************************
+   Auxiliar Functions
+ ****************************************/
+ 
+// cast from an array of chars to float value.
+float btof(byte * payload, unsigned int length) {
+  char * demo = (char *) malloc(sizeof(char) * 10);
```
Comment (jotathebest) [inline IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino:28]: Again, the memory reserved for this variable must be free

File: IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+void loop() {
+  // put your main code here, to run repeatedly:
+  if (!client.connected()) {
+    client.reconnect();
+    client.ubidotsSubscribe(DEVICE_LABEL, VARIABLE); //Insert the dataSource and Variable's Labels 
+  }
+  client.loop();
+}
```
Comment (jotathebest) [inline IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino]: add a new line

File: IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino (hunk 1 above)
Comment (mcmchris) [reply IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino:28]: Where exactly shoul I put the "free(demo);" code line?

Comment (jotathebest) [reply IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino:28]:
```
float btof(byte * payload, unsigned int length) {
  char * demo = (char *) malloc(sizeof(char) * 10);
  for (int i = 0; i < length; i++) {
    demo[i] = payload[i];
  }
  float value = atof(demo);
  free(demo);
  return value;
}
```

Comment (mcmchris) [reply IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino:28]: Thank you!

Comment (mcmchris) [reply IoT-Projects/ESP8266/Brushless Speed Controller/UBIDOTS_ESP8266_ESC-30A_BRUSHLESS_SPEED_CONTROLLER.ino:28]: Done!

**Pull request merged**